src/server/styles.test.jsx

```diff
@@ -31,7 +32,9 @@ describe('getStyleTag', () => {
       const inlineCss = renderToString(getStyleTag(sheet));
       expect(inlineCss).toBe(
         expectedOutput(
-          'data-styled="gDvQnu bKGMUB" data-styled-version="4.1.2"',
+          `data-styled="gDvQnu bKGMUB" data-styled-version="${
+            packageLock.dependencies['styled-components'].version
```

Comment:
Given the description of the test (`should respond with data-styled-component attribute`), this `data-styled-version` attribute feels like something of a red herring -- it feels like a lot of "work" for this test to be doing, & a piece of internal styled components working that we haven't got a clear reason to test? Could we ignore this attribute altogether, & focus on updating the test to _only_ test the `data-styled` attribute somehow?

I think we should focus on checking a) if the attribute is present, & b) that it's not `== ""`.

Also the test name seems out of date with what the attribute is called. It looks like it changed in v3 -> v4.

Reply:
I agree the title is incorrect, but I think the test is fine, we dont know what SC requires to work, if we somehow removed the data-styled-version attribute, we dont know that it would continue to work

Reply:
@bcmn @ChrisBAshton Not sure what you concluded in slack, what do you recommend for this test?

Reply:
I'm happy with this PR - and [thanks for renaming the test](https://github.com/bbc/simorgh/pull/1150/commits/ca8c039514891b41e7c8a7a364f8717f76809f90)